components/autocomplete/refresh-data.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Refresh Data
+page_title: Refresh Data
+description: Refresh Autocomplete Data using Observable Data or creating a new Collection reference.
+slug: autocomplete-refresh-data
+tags: telerik,blazor,autocomplete,observable,data,new,collection
+published: True
+position: 15
+---
+
+# Refresh Data
```

Comment:
Include the component name in the heading text, it will make it easier for people and google to distinguish one article from the other. Applies to all component-specific articles

Reply:
Added.